src/tfvc/commands/findworkspace.ts

```diff
@@ -113,15 +112,12 @@ export class FindWorkspace implements ITfvcCommand<IWorkspace> {
         }
         //If we're restricting the workspace, find the proper teamProject name
         if (this._restrictWorkspace) {
-            const folder: string = path.basename(this._localPath).toLowerCase();
-            //With tf.exe, folder and teamProject should match (TEE won't)
-            if (folder !== teamProject.toLowerCase()) {
-                for (let i: number = 0; i < mappings.length; i++) {
+            for (let i: number = 0; i < mappings.length; i++) {
```

Comment:
😄 Yeah, those are actually excluded by never being returned from the earlier call to `getMapping`.  TF.exe returns a cloaked mapping that ends with ':' and the CLC returns a cloaked mapping that doesn't end with ':'. Both of these fail the `if` in getMapping. We should talk about how to properly handle cloaked folders (since they aren't today).